src/main/java/org/HdrHistogram/StripedWriterReaderPhaser.java

```diff
@@ -199,25 +152,13 @@ private boolean caughtUp(final long[] startValueAtFlip, final AtomicLongArray en
 
     private int threadIndex() {
         final int index = (int) Thread.currentThread().getId();
-        return index & (stripes - 1);
+        return STRIDE * (index & (stripes - 1));
```

Comment:
Add a private final stripeMask field that is initialized to stripes - 1. This will eliminate an arithmetic operation from this hot path.

Reply:
Done